packages/dds/tree/src/tableSchema.ts

```diff
@@ -32,10 +32,14 @@ import {
 	type UnannotateImplicitFieldSchema,
 } from "./simple-tree/index.js";
 
-// Future improvement TODOs (ideally to be done before promoting these APIs to `@alpha`):
+// Future improvement TODOs:
```

Comment:
The remaining items here would be nice to have, but I don't think they're blocking.